app/code/Magento/Checkout/Block/Checkout/AttributeMerger.php

```diff
@@ -381,14 +381,13 @@ protected function getCustomer(): ?CustomerInterface
     /**
      * Retrieve field options from attribute configuration
      *
-     * @param string $attributeCode
      * @param array $attributeConfig
      * @return array
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    protected function getFieldOptions($attributeCode, array $attributeConfig)
+    protected function getFieldOptions(array $attributeConfig)
```

Comment:
Removing method argument is backward incompatible change. Please revert it

Reply:
sorry, I reverted it already.